admin/lib/solidus_admin/system/providers/main_nav.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "solidus_admin/container"
+require "solidus_admin/main_nav_item"
+
+module SolidusAdmin
+  module Providers
```

Comment:
Maybe a `Components` namespace would help here as well to understand the usecase of this "Provider" a bit better?

Reply:
See https://github.com/solidusio/solidus/pull/5099#discussion_r1207065776